argilla/src/argilla/records/_io/_datasets.py

```diff
@@ -98,64 +222,15 @@ def _get_image_features(dataset: "HFDataset") -> List[str]:
         Returns:
             bool: True if the Hugging Face dataset contains image features, False otherwise.
         """
-        media_features = [name for name, feature in dataset.features.items() if isinstance(feature, Image)]
-        return media_features
-
-    @staticmethod
-    def _cast_images_as_urls(hf_dataset: "HFDataset", columns: List[str]) -> "HFDataset":
-        """Cast the image features in the Hugging Face dataset as URLs.
-
-        Parameters:
-            hf_dataset (HFDataset): The Hugging Face dataset to cast.
-            repo_id (str): The ID of the Hugging Face Hub repo.
-
-        Returns:
-            HFDataset: The Hugging Face dataset with image features cast as URLs.
-        """
+        casted_features = defaultdict(list)
 
-        unique_identifier = uuid4().hex
+        for name, feature in hf_dataset.features.items():
+            if isinstance(feature, Image):
+                casted_features[Image].append(name)
+            if isinstance(feature, ClassLabel):
+                casted_features[ClassLabel].append(name)
```

Comment:
so, there is not differentiation between single/multi-label?

Reply:
`ClassLabel` == `rg.LabelQuestion` multi label in datasets is a `Sequence[ClassLabel]` but it's out of scope.